### PR TITLE
chore: minor adjustments

### PR DESCRIPTION
**Pull request by johanneswuerbach**
Minor adjustments, more details inline.

### CODE REVIEW

File: .github/workflows/close_pr.yaml [hunk 1]
```diff
@@ -15,10 +15,10 @@ jobs:
...
           version: ${{ env.HUMCTL_VERSION }}
-      
+
       - name: Delete Humanitec Env
         run: |
           humctl delete env pr-${{ github.event.number }} \
               --token ${{ secrets.HUMANITEC_TOKEN }} \
               --org ${{ vars.HUMANITEC_ORG }} \
-              --app ${{ vars.APP_NAME }}
+              --app ${{ vars.HUMANITEC_APP }}
```
Comment (johanneswuerbach) [inline .github/workflows/close_pr.yaml:24]: `HUMANITEC_APP` is the standard name to set your local context for humctl and the "name" is actually something different, hence I think using `HUMANITEC_APP` would be better.

File: .github/workflows/main.yaml [hunk 2]
```diff
@@ -4,6 +4,10 @@ on:
     branches:
       - "main"
 
+permissions:
+  contents: read
+  packages: write
```
Comment (johanneswuerbach) [inline .github/workflows/main.yaml:9]: This + using `${{ secrets.GITHUB_TOKEN }}` below avoids that people have to create a custom token and is generally more secure as those credentials are only short-living.

File: .github/workflows/main.yaml [hunk 3]
```diff
@@ -39,15 +43,18 @@ jobs:
           docker build . --tag $IMAGE  \
             --label "org.opencontainers.image.source=https://github.com/$GH_REPOSITORY"
           docker push $IMAGE
-          echo "IMAGE=$IMAGE" >> $GITHUB_OUTPUT
+
+          # Get image digest to pass on to deployment for pinning the image to that digest
+          IMAGE_DIGEST=$(docker inspect --format='{{index .RepoDigests 0}}' $IMAGE)
+          echo "IMAGE_DIGEST=$IMAGE_DIGEST" >> $GITHUB_OUTPUT
```
Comment (johanneswuerbach) [inline .github/workflows/main.yaml:49]: Previously any successful main deployment set `image:latest`, which actually doesn't roll the k8s pods as the image tag didn't change. Align with the digest used in the PR flow.

**Review by TobiasBabin — APPROVED** [review]
Great improvements, thanks a lot!

**Pull request merged**